ams/server/src/main/java/com/netease/arctic/server/table/executor/TableRuntimeRefreshExecutor.java

```diff
@@ -52,12 +52,16 @@ public void handleStatusChanged(TableRuntime tableRuntime, OptimizingStatus orig
   }
 
   private void tryEvaluatingPendingInput(TableRuntime tableRuntime, ArcticTable table) {
-    OptimizingEvaluator evaluator = new OptimizingEvaluator(tableRuntime, table);
-    if (evaluator.isNecessary()) {
-      OptimizingEvaluator.PendingInput pendingInput = evaluator.getPendingInput();
-      logger.debug("{} optimizing is necessary and get pending input {}", tableRuntime.getTableIdentifier(),
-          pendingInput);
-      tableRuntime.setPendingInput(pendingInput);
+    if (tableRuntime.isOptimizingEnabled() && !tableRuntime.getOptimizingStatus().isProcessing()) {
+      OptimizingEvaluator evaluator = new OptimizingEvaluator(tableRuntime, table);
+      if (evaluator.isNecessary()) {
+        OptimizingEvaluator.PendingInput pendingInput = evaluator.getPendingInput();
+        if (logger.isDebugEnabled()) {
```

Comment:
We no longer judge whether the debug mode is enabled.
reference: https://github.com/NetEase/arctic/issues/1570